.gitignore

```diff
@@ -126,3 +126,4 @@ modules/drivers/*/.lsp/*
 # jar downloader
 /jars/
 /bin/bb
+
```

Comment:
I just saw this. Hmmm. Nvm.

Reply:
ah, must've been when I removed the `.backup` from gitignore 😅